modules/nw-configuring-route-timeouts.adoc

```diff
@@ -7,26 +7,29 @@
 [id="nw-configuring-route-timeouts_{context}"]
 = Configuring route timeouts
 
-You can configure the default timeouts for an existing route when you
-have services in need of a low timeout, which is required for Service Level
-Availability (SLA) purposes, or a high timeout, for cases with a slow
-back end.
+You can configure the default timeouts for an existing route when you have services in need of a low timeout, which is required for Service Level Availability (SLA) purposes, or a high timeout, for cases with a slow back end.
+
+[IMPORTANT]
+====
+If you configured a user-managed external load balancer in front of your {product-title} cluster, ensure that the timeout value for the user-managed external load balancer is higher than the timeout value for the route service. This configuration prevents network congestion issues over the network that your cluster uses.
```

Comment:
The timeouts are set on routes, not services. I think that the wording should be slightly changed avoid the confusion:
```suggestion
If you configured a user-managed external load balancer in front of your {product-title} cluster, ensure that the timeout value for the user-managed external load balancer is higher than the timeout value for the route. This configuration prevents network congestion issues over the network that your cluster uses.
```